app/lib/discount.rb

```diff
@@ -0,0 +1,27 @@
+module Discounts
+
+  def discount_total
+    discount = 0
+    discount += discount_total_price
+    discount += discount_muffins
+    return (discount).round(2)
+  end
+
+  def discount_total_price
+    discount_level = 5
+    discount_applied_at = 50
+    net_total > discount_applied_at ? calculate_discount(net_total, discount_level) : 0
+  end
+
+  def discount_muffins
+    discount_level = 10
+    matched_order_lines = order.select {|line| line[:item].include?('Muffin')}
+    matched_total = matched_order_lines.inject(0){ | memo, n| memo + line_price(n) }
```

Comment:
Line is too long. [84/80]<br>Space before first block parameter detected.<br>Space missing to the left of {.